### DISCUSSION TITLE
How to start and running  conductor server

### DISCUSSION THREAD

**Discussion by hot740** [Netflix/conductor#2219] [Q&A] [answered]
I have clone the conductor source code. but I donnot konw how to run the conductor.am I just need to run conductor-server?
if so, do I need to run the client?

**Comment by aravindanr** [comment, marked as answer]
To run Conductor, you can simply run  the main method in `com.netflix.conductor.Conductor`.  You can also use Docker to run it. Instructions [here](https://github.com/Netflix/conductor/tree/main/docker#docker)
